PTA/pta.cpp

```diff
@@ -408,6 +456,59 @@ void PTA::handlePriceCheckHotkey(uint32_t flag)
     m_blockHotkeys = false;
 }
 
+void PTA::handleWikiHotkey(uint32_t flag)
+{
+    if (m_blockHotkeys)
+    {
+        // Currently blocked
+        return;
+    }
+
+    if (m_pcTriggered)
+    {
+        // Another search triggered
+        return;
+    }
+
+    m_blockHotkeys = true;
+
+    // Check for PoE window
+    if (!pta::IsPoEForeground())
+    {
+        m_blockHotkeys = false;
+        return;
+    }
+
+    // Arm trigger
+    m_pctype      = flag;
+    m_pcTriggered = true;
+
+    QTimer::singleShot(100, [=]() {
+        // Reset trigger if still armed
+        if (m_pcTriggered)
+        {
+            m_pcTriggered = false;
+        }
+    });
+
+    // Send ctrl-c
+    std::vector<INPUT> keystroke;
+
+    // ensure ctrl/alt/c is up
+    keystroke.push_back(pta::CreateInput(VK_MENU, false));
+    keystroke.push_back(pta::CreateInput(VK_CONTROL, false));
+    keystroke.push_back(pta::CreateInput('C', false));
+
+    keystroke.push_back(pta::CreateInput(VK_CONTROL, true));
+    keystroke.push_back(pta::CreateInput('C', true));
+    keystroke.push_back(pta::CreateInput('C', false));
+    keystroke.push_back(pta::CreateInput(VK_CONTROL, false));
+
+    SendInput(keystroke.size(), keystroke.data(), sizeof(keystroke[0]));
+
+    m_blockHotkeys = false;
+}
+
```

Comment:
This doesn't look any different from `PTA::handlePriceCheckHotkey().` Should probably be removed entirely in favor of `PTA::handlePriceCheckHotkey()` and perhaps change the method name to be more generic.

Reply:
I removed handleWikiHotkey and renamed handlePriceCheckHotkey to handleItemHotkey to have a function that handles hotkey events with item interactions.